include/deal.II/base/parameter_handler.h

```diff
@@ -1654,6 +1662,12 @@ class ParameterHandler : public Subscriptor
    */
   std::unique_ptr<boost::property_tree::ptree> entries;
 
+  /**
+   * Map that stores whether an entry added to the parameter handler has
```

Comment:
```suggestion
   * A map that stores whether an entry added to the parameter handler has
```

Reply:
done in https://github.com/dealii/dealii/pull/9718/commits/394b9d581ff172d6d979fa04cc1b5c908f59e3e3